defaults/main.yml

```diff
@@ -4,11 +4,27 @@ prometheus_version: 2.6.1
 prometheus_config_dir: /etc/prometheus
 prometheus_db_dir: /var/lib/prometheus
 
-prometheus_web_listen_address: "0.0.0.0:9090"
+prometheus_web_listen_port: 9090
+prometheus_web_listen_address: "0.0.0.0:{{ prometheus_web_listen_port }}"
 prometheus_web_external_url: ''
 
 prometheus_storage_retention: "30d"
 
+prometheus_docker_install: false
+
+prometheus_docker_image: "quay.io/prometheus/prometheus"
```

Comment:
Variable should be either removed or placed in `vars/main.yml`

Reply:
I agree with the policy because it makes things more clear as the official source is indeed only this one. I think having the possibility to overload that is convinient but I get your point too ! :)